README.rst

```diff
@@ -2,6 +2,46 @@
 ts_aos_utils
 ############
 
+Requirements
+------------
+
+This package requires some extra repositories to be cloned locally so it can grab some constants and some look-up tables. Here is a list of which repos are required to run this notebook:
+
+[ts_config_mttcs](https://github.com/lsst-ts/ts_config_mttcs)
+[ts_criopy](https://github.com/lsst-ts/ts_criopy)
+[ts_m2com](https://github.com/lsst-ts/ts_m2com)
+[ts_tcpip](https://github.com/lsst-ts/ts_tcpip)
+[ts_utils](https://github.com/lsst-ts/ts_utils)
+[ts_xml](https://github.com/lsst-ts/ts_xml)
+
+Since every user has a different setup, the paths might be slightly different. It is recommended to have all the repositories cloned under $HOME/notebooks. You might end up with many repositories and adding an extra folder with the name of the organization they belong might help to find them on GitHub later. For example, if you are working under notebooks_vandv environment, you should clone and install packages in the folder $HOME/notebooks/lsst-ts. The paths below consider this directory structure but, of course, you are free to organize your folders as you please.
```

Comment:
Please help to follow the Markdown syntax that each new sentence would be a new line. You can see that this will not change the format.

In addition, the path you mentioned only apply to the condition that people are using the RSP. You can run the notebook under your own laptop as well if you know how to do the related setup.